_inc/client/traffic/verification-services.jsx

```diff
@@ -16,11 +16,30 @@ import {
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
+import JetpackBanner from 'components/jetpack-banner';
 
 export const VerificationServices = moduleSettingsForm(
 	React.createClass( {
+		activateVerificationTools: function() {
+			return this.props.updateOptions( { [ 'verification-tools' ]: true } );
+		},
+
 		render() {
 			const verification = this.props.getModule( 'verification-tools' );
+
+			// Show one-way activation banner if not active
+			if ( ! this.props.getOptionValue( 'verification-tools' ) ) {
+				return (
+					<JetpackBanner
+						callToAction={ __( 'Activate' ) }
+						title={ verification.name }
+						icon="cog"
+						description={ verification.long_description }
+						onClick={ this.activateVerificationTools }
+					/>
```

Comment:
Yup, it's coming from dops-comonents and it seems like it's on purpose to include two svgs (GridIcons) here 

https://github.com/Automattic/dops-components/blob/master/client/components/banner/index.jsx#L76-L85